Shocksupershaman - Copy/src/main/java/com/example/shocksupershaman/shocktask1/UserService.java

```diff
@@ -0,0 +1,42 @@
+package com.example.shocksupershaman.shocktask1;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.Collectors;
+
+@Data
+@NoArgsConstructor
+@Service
+public class UserService {
+    private final Map<Long, User> db = new HashMap<>();
+    private static final AtomicLong COUNTER = new AtomicLong();

```

Comment:
Static fields declared first  before non static.
Atomic is used for another purposes. For random id generation use UUID.generate().toString()